.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+We are always happy to receive pull requests.
+
+Please make sure you have read our coding guidelines: https://github.com/mlr-org/mlr/wiki/mlr-Coding-Guidelines
```

Comment:
Lets brainstorm!
* code style
  * most obvious deviation from widespread R usage elsewhere is using `=` instead of `<-`
  * functions use camelcase, variables use dotted case
  * use `thirdparty/quicklint` to quickly check for coding guidelines
* tests
  * new functionality should be tested
  * bugfixes should have a test that catch the old bug, if reasonable
I'd assume most people who are at peril of submitting PRs to us (1) know how to use git and (2) (now that we don't rebase any more) know our process of doing PRs, so maybe add an encouraging "if you just do a small fix this is probably all you need to know"